e2e/tests/notebook/notebook-writer-actions.spec.ts

```diff
@@ -172,18 +117,48 @@ xdescribe('Workspace WRITER Jupyter notebook action tests', () => {
     const linkDisplayed = await workspaceLink.isVisible();
     expect(linkDisplayed).toBe(true);
 
-    // Verify copied notebook exists in collaborator Workspace.
-    notebookCard = await dataResourceCard.findCard({ name: copyNotebookName, cardType: ResourceCard.Notebook });
-    expect(notebookCard).toBeTruthy();
+    await verifyWriterNotebookPermission(page, notebookCopyName);
 
-    // Notebook actions Rename, Duplicate, Delete and Copy to another Workspace actions are available for click.
-    const copyNotebookCardMenu = await notebookCard.getSnowmanMenu();
-    expect(await copyNotebookCardMenu.isOptionDisabled(MenuOption.Rename)).toBe(false);
-    expect(await copyNotebookCardMenu.isOptionDisabled(MenuOption.Duplicate)).toBe(false);
-    expect(await copyNotebookCardMenu.isOptionDisabled(MenuOption.Delete)).toBe(false);
-    expect(await copyNotebookCardMenu.isOptionDisabled(MenuOption.CopyToAnotherWorkspace)).toBe(false);
-    await notebookCard.clickSnowmanIcon(); // close menu
+    // Cannot copy notebook to another workspace which already has a notebook with same name.
+
+    // Open Copy modal
+    const resourceCard = new DataResourceCard(page);
+    notebookCard = await resourceCard.findCard({ name: notebookCopyName, cardType: ResourceCard.Notebook });
+    await notebookCard.selectSnowmanMenu(MenuOption.CopyToAnotherWorkspace, { waitForNav: false });
+    // Fill out modal fields.
+    const copyModal = new CopyToWorkspaceModal(page);
+    await copyModal.waitForLoad();
+    await copyModal.copyToAnotherWorkspace(workspaceName, notebookName);
+    await waitWhileLoading(page);
 
-    await newAnalysisPage.deleteResource(copyNotebookName, ResourceCard.Notebook);
+    expect(await copyModal.isLoaded()).toBe(true);
+    modalText = await copyModal.getTextContent();
+    const regex = new RegExp(/Notebook with the same name already exists in the targeted workspace/i);
+    expect(modalText.some((text) => regex.exec(text) !== null)).toBe(true);
+    // Close Copy modal
+    await copyModal.clickButton(LinkText.Close, { waitForClose: true });
```

Comment:
Add new test coverage: Cannot copy notebook to another workspace if that workspace already has a notebook with same name.